CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# Release 2.21.0 02-12-2020
+
+- b/173810289: fix request header size (#425)
```

Comment:
- Fix request header size (#425)
- Update envoy to  jwt clock_skew change (#420)
- Support AuthenticationRule.allow_without_credential (#419)
- Enable traceparent trace context propagation by default (#416)
- Envoy changes to use DependencyErrorBehavior in TokenSubscriber (#406)

Reply:
Done. Thanks!